src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -9,14 +9,16 @@ namespace System.Diagnostics
 {
     public partial class Process
     {
+        private const int NanoSecondsTo100NanosecondsFactor = 100;
```

Comment:
It still says NanoSeconds :)

Reply:
🤦‍♂️ ...yupp... (it's very late here already) 
Will clean up - after a sleep.